docs/gambit/mutation-verifier.md

```diff
@@ -35,129 +34,193 @@ If you are on Linux, you will need pip version 20.3 or above.
 
 ## Running the Mutation Verifier
 
-Once you have updated your `certora-cli` installation using `pip` to get the relevant
-dependencies, run Gambit from the command line:
+The mutation verification script is called `certoraMutate`. 
+Run it from the command line:
 
-```
-certoraMutate --prover_conf path/to/prover/prover.conf --mutation_conf path/to/mutation/mutation.conf
+```sh
+certoraMutate --prover_conf path/to/prover.conf --mutation_conf path/to/mutation.mconf
 ```
 
 ```{note}
 You must run `certoraMutate` from the root of the Solidity project directory.
-The files `prover.conf` and `mutation.conf`
-can be in their own directories, but must always be within the project directory.
+The files `prover.conf` and `mutation.conf` can be in their own directories, 
+  but must always be within the project directory.
 All paths in `mutation.conf` are relative to the parent directory containing `mutation.conf`.
-Paths in `prover.conf` are all relative to the project directory's root, which is assumed to be the working directory.
+Paths in `prover.conf` are all relative to the project directory's root, 
+  which is assumed to be the working directory.
 ```
 
-## Configurations
-The tool expects two separate configuration files:
-the configuration file which defines the execution of mutant generation (`--mutation_conf`),
-and the configuration file which defines execution of the Prover (`--prover_conf`).
-Here is a simple configuration file setup using the example above:
-
-In `prover.conf`:
+## Mutation Configuration
+The tool expects a configuration file which defines the execution of 
+  mutant generation (`--mutation_conf`).
+Here is a simple configuration file setup using the example above in `mutation.mconf`:
 
 ```json
-{
-  "files": [
-    "C.sol"
-  ],
-  "verify": "C:c.spec"
-}
-```
-In `mutation.conf`:
-
-```json
-{ "gambit": [{
+{ 
+  "gambit": [{
     "filename" : "C.sol",
     "num_mutants": 5
   }]
 }
 ```
 
+### Automatic Generation
+You can generate a mutation configuration automatically from a Prover configuration 
+  with the flag `--generate_mutation_conf`.
+The resulting configuration may need some manual path fixing or the addition of other
+ `certoraMutate` specific options.
+
+Example usage:
```

Comment:
should we show the output?

Reply:
The entire output is too long, but I added the final two lines that indicate success and a screenshot of the email that you should get.